resources/debugger/py_debug_wrapper.py

```diff
@@ -1,14 +1,34 @@
 import os
 import runpy
 import sys
+import re
+
+# gives us access to debugpy's log information so we can emit a 'wait_for_client()' message
+class Pipe:
+    def write(self, message):
+        # debugpy log messages look like this I+xxxxx.xxx: where x is a number
+        # we don't want to pipe those to stderr as it will clutter the debug console
+        # however, we do want to output actual errors since it is very useful for debugging
+        if 'wait_for_client' in message:
+            print('Debugger waiting for client...')
+            self.flush()
+        elif not re.search('(I|D)\+[0-9]+\.[0-9]+:', message):  
+            # not a log message, print it
```

Comment:
can you show an example of what the regex is supposed to match:
```suggestion
            # Log message looks like:
            #     D99: foo
            # Print all other messages.
```

Reply:
There's a comment further up, but I'll move it to prevent confusion. User wouldn't want to see those messages. I can add a '--debug' argument to the wrapper for our use though.